provider/datagen/tests/test-baked-source.rs

```diff
@@ -0,0 +1,49 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+extern crate alloc;
+
+mod testutil {
+    include!("testutil.rs");
+}
```

Comment:
nit: nicer way to include a whole file as a module
```suggestion
#[path = "testutil.rs"]
mod testutil;
```

Reply:
done